src/core/execute.c

```diff
@@ -445,8 +443,12 @@ int exec_spawn(Unit *unit,
         if (r < 0)
                 return log_unit_error_errno(unit, r, "Failed to get executor path from fd: %m");
 
+        char serialization_fd_number[DECIMAL_STR_MAX(int)];
```

Comment:
This had `+1` above but now it's gone, if that's intentional the commit message should mention it, right now it only says `core/execute: re-break function declaration of exec_spawn`